doc/reference/proxysource.rst

```diff
@@ -0,0 +1,24 @@
+.. _ProxySource:
+
+ProxySource
+===========
+
+Base interface for all supported sources in :ref:`ProxySChunk`.
+
+In case the source is multidimensional, the attributes `shape`, `chunks`,
+`blocks` and `dtype` are also required when creating the :ref:`ProxySChunk`.
+In case the source is unidimensional, the attributes `chunksize`, `typesize`
+and `nbytes` are required as well when creating the :ref:`ProxySChunk`.
+These attributes do not need to be available when opening an already
+existing :ref:`ProxySChunk`.
```

Comment:
This docstring is duplicated from `ProxySChunk` description.  Could sphinx be instructed to insert the description here?

Reply:
It is possible but the structure and design of attributes, methods, etc, will be different from the other classes.